### PR TITLE
Update crossbeam to 0.5 fixed a bug in index creation causing a panic

### PR DESCRIPTION
**Pull request by hntd187**

### CODE REVIEW

File: src/handlers/bulk.rs [hunk 1]
```diff
@@ -62,11 +62,14 @@ impl Handler for BulkHandler {
             thread::spawn(move || {
```
Comment (LucioFranco) [inline src/handlers/bulk.rs]: Have you thought about using rayon at all for this?

Comment (LucioFranco) [reply src/handlers/bulk.rs]: I have yet to really use it but I know that if we are processing things in parallel pretty much allows you to parallelize iterators etc. Just might be something to look into down the line.

File: src/handlers/bulk.rs [hunk 2]
```diff
@@ -88,13 +91,19 @@ impl Handler for BulkHandler {
                     if split.peek().is_none() {
                         return future::ok(l.to_vec());
                     }
-                    line_sender_clone.send(l.to_vec());
+                    match line_sender_clone.send(l.to_vec()) {
+                        Ok(_) => (),
+                        Err(e) => return future::err(e.into_handler_error()),
```
Comment (LucioFranco) [inline src/handlers/bulk.rs]: You dont need the return here, you can just let the match return and have the error fall through I believe. the other thing too is that you can just `map` and `map_err` these futures.

Comment (hntd187) [reply src/handlers/bulk.rs]: The match is in a while loop so I don't think that will work unless I'm missing something?

Comment (LucioFranco) [reply src/handlers/bulk.rs]: Yeah, I see what you are saying, I probably would have used an iterator there instead of while loops but its fine.

Comment (hntd187) [reply src/handlers/bulk.rs]: I tried to do that and eventually gave up I couldn't get the iterators to play nice with me there at all.

Comment (LucioFranco) [reply src/handlers/bulk.rs]: Yeah, that's too bad

**Pull request merged**